.pre-commit-config.yaml

```diff
@@ -82,11 +82,12 @@ repos:
   - repo: local
     hooks:
       - id: pylint_warnings
-        args: ["--rcfile=.pylintrc"]
+        args: ["--rcfile=.pylintrc.precommit.ini"]
         name: pylint-warnings
         entry: python -m pylint
         language: system
         files: \.py$
+        exclude: clarity/_version.py # 3rd party code that doesn't conform
```

Comment:
I'd hoped that #186 would help here but `setuptools_scm` doesn't (yet^1) produce a Black compliant `_version.py` as it uses `'` rather than `"` see [setuptools_scm 482](https://github.com/pypa/setuptools_scm/issues/482))

^1 I had a brief attempt at addressing that but its more involved than I thought, will get back to it somewhen.